engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -4128,6 +4129,7 @@ public void mainHelper() throws Exception
 
   public static void main(final String[] args) throws Exception
   {
+    LoggerUtil.addAppender("apexclient");
```

Comment:
Vlad suggested to add the new method getLongName() to the class Attribute and use the method getLongName() instead of the method getName() as a name of the command line parameter for setting of the appender properties. The log name of the attribute LOGGER_APPENDER is "apex.logger.appender". Please see the implementation below.

  public String getLongName()
  {
    return "apex." + getSimpleName().replaceAll("_",".").toLowerCase();
  }